src/Markdig/MarkdownPipelineBuilder.cs

```diff
@@ -80,7 +80,7 @@ public MarkdownPipelineBuilder()
         /// True to parse trivia such as whitespace, extra heading characters and unescaped
         /// string values.
         /// </summary>
-        public bool TrackTrivia { get; set; }
+        internal bool TrackTrivia { get; set; }
```

Comment:
How to setup the trivia on builder if we can't access it from the public API?

Probably needs an extension method on the builder e.g `WithTrivia()`

Reply:
There is a `EnableTrackTrivia` extension exposed, but it does more than just set this bool. The actual behavior can't be toggled on/off with just this bool, so having it introduces ambiguity where only the extension method should be used.

Reply:
Fair enough, but maybe make it public getter still - as you might want to develop an extension that can react to a previously setup track trivia.

Reply:
Changed to

```c#
public bool TrackTrivia { get; internal set; }
```